Assets/Scripts/Content/QuestData.cs

```diff
@@ -1179,8 +1179,8 @@ override public string ToString()
     public class Puzzle : Event
     {
         new public static string type = "Puzzle";
-        public string puzzleClass = "slide";
-        public string skill = "{observation}";
+        public StringKey puzzleClass = new StringKey("val", "PUZZLE_SLIDE_CLASS");
```

Comment:
This is a magic string, I am not sure that it should be translated.  At most perhaps translate it in the editor display, but keep it as a normal string internally.